### PR TITLE
Add Pester framework for automated testing of the Powershell layer

### PR DESCRIPTION
**Pull request by abhi18av**
I've added the 
- `Pester` framework for testing the Powershell layer 
- Code in F# layer to manage the input validation

You can create a configuration in Rider which can automatically execute Pester tests

image

### CODE REVIEW

File: AddSevenPS/Library.fs [hunk 1]
```diff
@@ -17,10 +17,12 @@ module AddSevenPS=
         [<Parameter(Position = 0)>]
         member val Num : string = "" with get , set
         override this.EndProcessing() =
+                 
+            let mutable result = 0
```
Comment (abhi18av) [inline AddSevenPS/Library.fs:21]: @Aryan-sharma11 , notice the use of mutable variable here and how this is then used in Line-23 (via `&result`) to store the parsed number upon success.

There are better ways to manage this, such as exception and active-patterns - but I leave that task of refactoring this to you when you have gone through the F# codebases and manuals and have an "expert" opinion on how this should be done.

File: PesterTests/Add-Seven.Tests.ps1 [hunk 2]
```diff
@@ -1,5 +1,5 @@
 BeforeAll {
-    Import-Module ../AddSevenPS/bin/Debug/net6.0/AddSevenPS.dll
+    Import-Module ../AddSevenPS/bin/Debug/net6.0/publish/AddSevenPS.dll
```
Comment (abhi18av) [inline PesterTests/Add-Seven.Tests.ps1:2]: Good change! 👍

**Comment by abhi18av** [comment]
@Aryan-sharma11 , do you think it looks good to merge now? 

If so, please take this forward.

**Comment by Aryan-sharma11** [comment]
> @Aryan-sharma11 , do you think it looks good to merge now?
> 
> If so, please take this forward.

Yes, it looks fine to me

**Pull request merged**